Command/ResolveCacheCommand.php

```diff
@@ -21,68 +22,109 @@
+  <info>- bar.ext[thumb1] status: http://localhost/media/cache/thumb1/bar.ext</>
+
+<comment># bin/console %command.name% --filter=thumb1 --filter=thumb3 foo.ext</comment>
+Resolve <comment>foo.ext</comment> image using <options=bold>two</> filters (<comment>thumb1</comment> and <comment>thumb3</comment>), outputting:
+  <info>- foo.ext[thumb1] status: http://localhost/media/cache/thumb1/foo.ext</>
+  <info>- foo.ext[thumb3] status: http://localhost/media/cache/thumb3/foo.ext</>
+
+<comment># bin/console %command.name% foo.ext</comment>
+Resolve <comment>foo.ext</comment> image using <options=bold>all</> filters (as none were specified), outputting:
+  <info>- foo.ext[thumb1] status: http://localhost/media/cache/thumb1/foo.ext</>
+  <info>- foo.ext[thumb2] status: http://localhost/media/cache/thumb2/foo.ext</>
+  <info>- foo.ext[thumb3] status: http://localhost/media/cache/thumb2/foo.ext</>
+
+<comment># bin/console %command.name% --force --filter=thumb1 foo.ext</comment>
+Resolve <comment>foo.ext</comment> image using <options=bold>one</> filter (<comment>thumb1</comment>) and <options=bold>forcing resolution</> (regardless of cache), outputting:
+  <info>- foo.ext[thumb1] resolved: http://localhost/media/cache/thumb1/foo.ext</>
+
 EOF
             );
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $paths = $input->getArgument('paths');
-        $filters = $input->getOption('filters');
+        $this->setupOutputStyle($input, $output);
+        $this->outputCommandHeader();
+
+        $forced = $input->getOption('force');
+        [$images, $filters] = $this->resolveInputFiltersAndPaths($input);
+
+        foreach ($images as $i) {
+            foreach ($filters as $f) {
+                $this->runCacheImageResolve($i, $f, $forced);
+            }
+        }
+
+        $this->outputCommandResult($images, $filters);
+
+        return $this->getResultCode();
+    }
 
-        if (empty($filters)) {
-            $filters = array_keys($this->filterManager->getFilterConfiguration()->all());
+    private function runCacheImageResolve(string $image, string $filter, bool $forced): void
+    {
+        if (!$this->outputMachineReadable) {
+            $this->io->text(' - ');
         }
 
-        foreach ($paths as $path) {
-            foreach ($filters as $filter) {
-                $output->writeln($this->filterService->getUrlOfFilteredImage($path, $filter));
+        $this->io->group($image, $filter, 'blue');
+        $this->io->space();
+
+        try {
+            if ($forced || !$this->cacheManager->isStored($image, $filter)) {
+                $this->cacheManager->store($this->filterManager->applyFilter($this->dataManager->find($filter, $image), $filter), $image, $filter);
+                $this->io->status('resolved', 'green');
+            } else {
+                $this->io->status('cached', 'yellow');
```

Comment:
do we have something like "light green"? yellow indicates a warning, but existing cache is not a problem. maybe just white?

Reply:
``
Available foreground and background colors are: black, red, green, yellow, blue, magenta, cyan and white.
``
yep, white could be fine